js-contribution-guidelines.md

```diff
@@ -0,0 +1,12 @@
+# JS Contribution Guidelines
+
+Many IPFS projects use javascript. Please check these guidelines before contributing js code to an IPFS repository.
+
+## Suggestions
+
+- Build system: Use `npm run` for small projects; for larger ones, use `gulp`.
+- Testing node: `mocha`
+- Testing browser: `karma` + `mocha`
+- Browser building: `webpack` or `browserify` (this fight is still ongoing ;)
+- Linting: `eslint` + `eslint-config-standard`. This is roughly the same as [feross/standard](//github.com/feross/standard), but is easier to integrate and customize if needed.
```

Comment:
I suggest sth similar we have for `npm scripts` and `gulp. 
For simple projects use `standard` for more complex things you can use `eslint` with `eslint-config-standard`